src/nvmpi_dec.cpp

```diff
@@ -500,8 +505,7 @@ void dec_capture_loop_fcn(void *arg)
 	return;
 }
 
-//TODO: accept in nvmpi_create_decoder stream params (width and height, etc...) from ffmpeg.

```

Comment:
It's too early to delete this line. More options will need to be added in the future. .)